samples/snippets/webhook_prebuilt_telecom.py

```diff
@@ -0,0 +1,253 @@
+            'mexico',
+            'russia',
+            'singapore',
+        ]
+        # Sample list of covered international daily destinations.
+        covered_by_daily = [
+            'anguilla', 'australia', 'brazil', 'canada', 'chile', 'england',
+            'france', 'india', 'japan', 'mexico', 'singapore'
+        ]
+        if destination.lower() in covered_by_monthly and destination.lower() in covered_by_daily:
+            coverage = 'both'
+        elif destination.lower() in covered_by_monthly and destination.lower() not in covered_by_daily:
+            coverage = 'monthly_only'
+        elif destination.lower() not in covered_by_monthly and destination.lower() not in covered_by_daily:
+            coverage = 'neither'
+        else:
+            # This should never happen, because covered_by_daily is a subset of
+            # covered_by_monthly
+            coverage = 'daily_only'
+
+        res = {
+            "sessionInfo": {
+                "parameters": {
+                    "coverage": coverage,
+                    }
+                }
+            }
+
+    # BEGIN cheapestPlan
+    elif tag == 'cheapestPlan':
+        print(tag + ' was triggered.')
+        trip_duration = parameter_dict["trip_duration"]
+        monthly_cost = None
+        daily_cost = None
+        suggested_plan = None
+
+        # Can only suggest cheapest if both are valid for location.
+
+        # When trip is longer than 30 days, calculate per-month cost (example $
+        # amounts). Suggest monthly plan.
+        if trip_duration > 30:
+            monthly_cost = (int(trip_duration / 30)) * 70
+            daily_cost = trip_duration * 10
+            suggested_plan = 'monthly'
+
+        # When trip is <= 30 days, but greater than 6 days, calculate monthly
+        # plan cost and daily plan cost. Suggest monthly b/c it is the cheaper
+        # one.
+        elif trip_duration <= 30 and trip_duration > 6:
+            monthly_cost = 70
+            daily_cost = trip_duration * 10
+            suggested_plan = 'monthly'
+
+        # When trip is <= 6 days, calculate daily plan cost. Suggest daily
+        # plan.
+        elif trip_duration <= 6 and trip_duration > 0:
+            monthly_cost = 70
+            daily_cost = trip_duration * 10
+            suggested_plan = 'daily'
+
+        else:
+            # This should never happen b/c trip_duration would have to be
+            # negative
+            suggested_plan = 'null'
+
+        res = {
+            "sessionInfo": {
+                "parameters": {
+                        "monthly_cost": monthly_cost,
+                        "daily_cost": daily_cost,
+                        "suggested_plan": suggested_plan,
+                    }
+                }
+            }
+
+    # Default Case
+    else:
+        res = None
+        print(f'{"default case called"}')
+
+    # Returns json
+    return res
+
+
+# Dummy helper function
+def get_date_details(bill_month):
```

Comment:
I believe we got the implementation of this function from the buganizer request; can we add this in?

Reply:
Added :)